src/Compilers/CSharp/Test/Emit/CodeGen/UnsafeTests.cs

```diff
@@ -10328,10 +10328,13 @@ public static void Main()
     }
 }
 ";
-            // PEVerify:
-            // [ : ChannelServices::.cctor][mdToken=0x6000005][offset 0x0000000C][found unmanaged pointer][expected unmanaged pointer] Unexpected type on the stack.
-            // [ : ChannelServices::GetPrivateContextsPerfCounters][mdToken= 0x6000002][offset 0x00000002][found Native Int][expected unmanaged pointer] Unexpected type on the stack.
-            CompileAndVerify(text, options: TestOptions.UnsafeReleaseExe, verify: Verification.FailsPEVerify).VerifyDiagnostics();
+            CompileAndVerify(text, options: TestOptions.UnsafeReleaseExe, verify: Verification.FailsPEVerify with
+            {
+                PEVerifyMessage = """
```

Comment:
Consider also updating a test for ILVerifyMessage.

Reply:
IL verify does not report an error here.

Reply:
Any test where IL verify complains will do.

Reply:
I see what you mean